docs/touch-up.sh

```diff
@@ -11,10 +11,10 @@ for i in ls -l /docs/content/*
         y=${i##*/}
         find $i -type f -name "*.md" -exec sed -i.old \
         -e '/^<!.*metadata]>/g' \
+	-e '/(https:\/\//!s/(\([^)]*\.md\))/({{< relref "\1" >}})/g' \
```

Comment:
should this script be part of docs-base so that it can be used in the other repos as well?